types/seatsio/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
I guess the issue here is that [seatsio](https://www.npmjs.com/package/seatsio) is already taken on npm but these types refer to a different set of JS code?

Reply:
Correct.

These types are for the integration library. 
E.g. SeatingChart integration on a website.

The package on npm is (the js version of) the API Client, 
which is intended for a node-js backend.